### DISCUSSION TITLE
Checkout Pro: Getting the sandbox url to test payment

### DISCUSSION THREAD

**Discussion by rafaelfaria** [mercadopago/sdk-js#60] [Q&A] [answered]
Hi,

I just started integrating our platform with Mercado Pago and came across an issue.

We are using the Checkout Pro and we intend to run it as a modal and with `autoOpen: true` , however, during my testing, it opens the production (`init_point`) URL and I can't find a way in the documentation to tell the checkout to open the sandbox init point. 

Can someone help me? How do you test the checkout pro to open automatically with the sandbox page?

thanks

**Comment by eltinMeli** [comment, marked as answer]
Hello @rafaelfaria!

I hope you are well! In the mentioned scenario, just use your test credential which will automatically be redirected to the Sandbox environment. You can find it at our website https://www.mercadopago.com.br/developers/panel/credentials.

<img width="751" alt="image" src="https://user-images.githubusercontent.com/85578192/162740505-fec949a5-9c4c-4ad8-926a-deace8d27391.png">

Page Sandbox
<img width="1394" alt="image" src="https://user-images.githubusercontent.com/85578192/162741751-23498392-9281-4ece-8f0d-0a8f2b4e199d.png">

Get preferences
<img width="321" alt="image" src="https://user-images.githubusercontent.com/85578192/162741882-23ec4f26-4de9-44eb-b1a6-a847cec03b62.png">


If you persist with the problem, I suggest you open a ticket at https://www.mercadopago.com.br/developers/en/support/center/tickets. Our team will be happy to assist you.

Hugs!

**Reply by rafaelfaria** [reply]
Nice @eltinMeli. 

I couldn't find it in the documentation but I assumed that using the "Credenciais de tested" would give me the sandbox URL, and when i move it to prod and use its credential, it would be using the production URL. 

Appreciate the help.
